src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -282,45 +214,50 @@ public boolean isConnectedToComponent(Set<LfBus> connectedComponent) {
     }
 
     static class LfBranchFlowPerPSTAngle extends LfBranchPerPSTAngle {
```

Comment:
Ideally we should also refactor LfFactor subclasses to reflect new more generic API

Reply:
Yep, this would be great, do you want to do it within this PR, or another one ?

Reply:
This PR is already very large, I think we should do it in another PR. Any idea how to do it?